.github/workflows/main.yml

```diff
@@ -7,7 +7,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        ghc: ['8.6', '8.8', '8.10']
+        ghc: ['8.8', '8.10']
```

Comment:
@cptrodolfox if older versions of the compiler are causing issues, I'm fine about dropping support as long as we document that in the `.cabal`. Would you mind updating the `tested-with` entry as well, please?

Reply:
Adding support for GHC 9.x would be nice, however, I think we could tackle that in a separate PR :)

Reply:
Sure, I will remove the latest version of GHC from the matrix on the workflow. I will be updated the tested-with in the .cabal files. One question, Should we keep the stack.yaml and package.yaml files on this project if we are using cabal as the building tool?

Reply:
@cptrodolfox keeping both package managers has some pros and cons, I'll like to discuss that deeper before taking a final decision, in the meantime I think we could preserve those files since the scope of this PR mainly covers upgrading `aeson`. What do you think?

Reply:
Yes it sounds good. Also, I just discovered that we use the package.yaml for one of the tests inside ClientSpec.hs. We might need to study a better approach for defining the package version of the default notifier inside Rollbar.Client.Item.